helm-chart/binderhub/values.yaml

```diff
@@ -266,6 +267,22 @@ dind:
   hostSocketDir: /var/run/dind
   hostLibDir: /var/lib/dind
 
+# Podman in Kubernetes
+pink:
+  initContainers: []
+  daemonset:
+    image:
+      name: quay.io/podman/stable
+      tag: v4.2.0
```

Comment:
A new image is added as a dependency, can you add automation to automatically bump it? We have a workflow to do that, and it may be straight forward.

See https://github.com/jupyterhub/binderhub/blob/992cac3844a078534f02aea8940b6e8d246ba6be/.github/workflows/watch-dependencies.yaml#L28-L31 as we have it for dind, give it your best shot and I can help fix eventual bugs following it.

Reply:
There's a bit of an inconsistency between the comment and the actual `jq` query used with regard to the `v` handling. That would cause the use of an older version of the `podman` image as they have one that is tagged without `v`. 

What about using `jq -r '[.Tags[] | match("^v?(?<version>\\d+\\.\\d+\\.\\d+)$").captures[].string ] | sort_by(split(".") | map(tonumber)) | last'` ?

Reply:
Yes maybe, could need a strip of a v when sorting versions etc to not avoid erroring on mapping to number.

Dont want this to be a blocker for you, i can resolve this later.

Reply:
There are likely better (and cleaner) ways to do that as I am not that proficient with json query. I just saw the issue because there's an inconsistency in the tagging of the Podman stable images. There's both a `v3.4.4` and `3.4.4` that are one month apart and  pointing to different hashes. When testing that I had added the correct information in `watch-dependencies.yaml` I saw that it would select `3.4.4` rather than the latest of the 4 series.

Reply:
Your example led to 4.3.1 instead of v4.3.1. The following change did the trick though. From the current line with `jq`, I added `v?` to the match pattern, and `ltrimstr("v") |` before calling `tonumber` during the sorting procedure.

```
            | jq -r '[.Tags[] | select(. | match("^v?\\d+\\.\\d+\\.\\d+$") | .string)] | sort_by(split(".") | map(ltrimstr("v") | tonumber)) | last'
```